### ISSUE TITLE
Volume problem after noise reduction

### ISSUE THREAD

**Issue by zoujh320** [haoxiangsnr/FullSubNet#31]
Sometimes there are volume variations when speech is enhanced by pre-trained model，for example：
noisy：
![image](https://user-images.githubusercontent.com/31227658/141231693-c17a964b-0e32-437a-995b-c89b7d74b202.png)
enhanced：
![image](https://user-images.githubusercontent.com/31227658/141231804-15398a49-1b9c-472d-91c5-8d1b505ad17e.png)

**Comment by haoxiangsnr** [comment]
Hi, 

In the DNS challenge test dataset, there are many speeches with low sound volume. If we directly listen to the results enhanced by neural networks, the volume of different speeches is not equal and uncontrolled, i.e., it could be higher or lower than the original input. We may need to constantly change the volume of speech to adapt to different volume levels during the hearing test.

For this reason, in FullSubNet, we normalize the speech volume https://github.com/haoxiangsnr/FullSubNet/blob/main/audio_zen/inferencer/base_inferencer.py#L177.  In this way, the resulting speeches will have comparable sound volume.

**Comment by zoujh320** [comment]
Hi，
This just sidesteps the volume issue, how can I use the FullSubNet model to reduce the noise without changing the volume.
